com.unity.render-pipelines.high-definition/Documentation~/Override-Indirect-Lighting-Controller.md

```diff
@@ -17,10 +17,11 @@ The **Indirect Lighting Controller** uses the [Volume](Volumes.html) framework,
 
 | Property                        | Description                                                  |
 | ------------------------------- | ------------------------------------------------------------ |
-| **Indirect Diffuse Intensity**  | A multiplier for baked and realtime Global Illumination lightmaps and Light Probes. HDRP multiplies the lightmap and Light Probe data by this value. |
-| **Indirect Specular Intensity** | A multiplier for baked, realtime, and custom Reflection Probes. HDRP multiplies the Reflection Probe data by this value. |
-
-
+| **Indirect Diffuse Lighting Multiplier**  | A multiplier for lightmaps, Light Probes, Light Probe Volumes, Screen Space Global Illumination and Raytrace Global Illumination. HDRP multiplies all those data by this value. |
+| **Indirect Diffuse Lighting Layers** | Light layers for indirect diffuse lighting. See [LightLayers](Light-Layers.md). After you enable Light Layers, you can then use them to decouple Meshes from the this multiplier in your Scene. |
```

Comment:
Specifies the [Light Layers](Light-Layers.md) for indirect diffuse lighting. If you enable Light Layers, you can use them to decouple Meshes in your Scene from the above multiplier.